tests/unit/certGen.unitTest.ts

```diff
@@ -45,6 +52,16 @@ describe("cert-gen", () => {
         sandbox.restore();
         jest.setTimeout(5000);
     });
+    beforeEach(() => {
```

Comment:
Is this needed in a before each here? I think this is asking for issues later down the line, doing this for every test here